wcfsetup/install/files/lib/acp/form/PageAddForm.class.php

```diff
@@ -740,7 +730,8 @@ public function assignVariables() {
 			'title' => $this->title,
 			'content' => $this->content,
 			'metaDescription' => $this->metaDescription,
-			'metaKeywords' => $this->metaKeywords,
+			// @deprecated 5.4 - Meta keywords have been removed with 5.4.
```

Comment:
```suggestion
			/** @deprecated 5.4 - Meta keywords have been removed with 5.4. */
```
Reason: PHPDoc comments are defined to start with `/**`